listener/src/main.rs

```diff
@@ -247,5 +276,54 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
             .expect("Failed to start GraphQL subscription server"),
     );
 
+    // // Periodically check for contention in the tokio threadpool. First spawn a
```

Comment:
Is this needed?

Reply:
It isn't needed but it might be worth adding, this is what I mentioned here:
> - Somewhere between v0.13.2 and v0.16.1, an extra ping loop was added to check the health of the `tokio` executor that drives the async `Future`s. It might be worth including in the listener. It is currently commented out at the end of `async_main`.